src/app/profile/overview/spaces/spaces.component.ts

```diff
@@ -1,12 +1,11 @@
-import { Component, OnDestroy, OnInit, ViewEncapsulation, ViewChild } from '@angular/core';
+import {Component, OnDestroy, OnInit, ViewEncapsulation, ViewChild, TemplateRef} from '@angular/core';
```

Comment:
Another place that needs white space at beginning and end. I also think this list is long enough it could due with being a multi-line.

Reply:
what is the rule for having a multi-line import?